### ISSUE TITLE
Add documentation on enabling login capability

### ISSUE THREAD

**Issue by middleagedman** [diginc/docker-pi-hole#99]
First off awesome build and documentation!  I can finally repurpose my RPi and use my Arch Linux server for DNS.

FYI.. You may want to add to your document that you can enable the cool login page by running:

docker exec pihole_container_name sudo pihole -a -p \<password\>

Then your users won't be able to muck with your configuration if they do go to the pi.hole web page.

**Comment by rosshettel** [comment]
It would be great if this could be included as a environment variable so the password would persist on container restarts.

**Comment by middleagedman** [comment]
You know.. strange thing is I think it is stored somewhere.  I have my /etc/pihole directory mounted from my container, and I've destroyed and rerun the image many times, and somehow that password still sticks.

**Comment by Mcat12** [comment]
It's stored in setupVars.conf as a hash.

**Comment by diginc** [comment]
@rosshettel I've debated having a default random password like the new pi-hole installer does.  I ended up leaning towards these implementation details:

* Default: random password prints during the 1st boot of container
 * Users check their docker log for the line that says 'password set to' to find new random password.
 * Users could just override with a `docker exec pihole pihole -a -p <foobar>`
* env var `web_password`: when null / empty nullifies the need for a password
 * not null value sets the password to that value on first boot.
